src/command/data.rs

```diff
@@ -555,18 +555,32 @@ fn cardholder_cert<const R: usize, T: trussed::Client>(
 }
 
 fn pw_status_bytes<const R: usize, T: trussed::Client>(
-    mut ctx: LoadedContext<'_, R, T>,
+    mut ctx: Context<'_, R, T>,
 ) -> Result<(), Status> {
-    let status = PasswordStatus {
-        pw1_valid_multiple: ctx.state.internal.pw1_valid_multiple(),
-        max_length_pw1: MAX_PIN_LENGTH as u8,
-        max_length_rc: MAX_PIN_LENGTH as u8,
-        max_length_pw3: MAX_PIN_LENGTH as u8,
-        error_counter_pw1: ctx.state.internal.remaining_tries(Password::Pw1),
-        // TODO when implementing RESET RETRY COUNTER
-        error_counter_rc: 3,
-        error_counter_pw3: ctx.state.internal.remaining_tries(Password::Pw3),
+    let status = if let Ok(ctx) = ctx.load_state() {
+        PasswordStatus {
+            pw1_valid_multiple: ctx.state.internal.pw1_valid_multiple(),
+            max_length_pw1: MAX_PIN_LENGTH as u8,
+            max_length_rc: MAX_PIN_LENGTH as u8,
+            max_length_pw3: MAX_PIN_LENGTH as u8,
+            error_counter_pw1: ctx.state.internal.remaining_tries(Password::Pw1),
+            // TODO when implementing RESET RETRY COUNTER
```

Comment:
I just noticed this to do – we’ve implement RESET RETRY COUNTER but forgot to use the real value here?

Reply:
Fixed